showcase/examples/force-directed-graph/force-directed-graph.js

```diff
@@ -99,10 +99,12 @@ class ForceDirectedGraph extends React.Component {
     };
   }
 
-  componentWillReceiveProps(nextProps) {
-    this.setState({
-      data: generateSimulation(nextProps)
-    });
+  componentDidUpdate(prevProps, prevState, snapshot) {
```

Comment:
why snapshot parameter is declared if it's not used inside componentDidUpdate. Remove unused parameters.